docs/examples.rst

```diff
@@ -401,10 +401,10 @@ If you need a codec different from ASCII or UTF-8, you can use
 Sorting a Pandas DataFrame
 --------------------------
 
-As of Pandas version 0.16.0, the sorting methods do not accept a ``key``
-argument, so you cannot simply pass :func:`natsort_keygen` to a Pandas
-DataFrame and sort. This request has been made to the Pandas devs; see
-`issue 3942 <https://github.com/pydata/pandas/issues/3942>`_ if you are interested.
-If you need to sort a Pandas DataFrame, please check out
-`this answer on StackOverflow <https://stackoverflow.com/a/29582718/1399279>`_
-for ways to do this without the ``key`` argument to ``sort``.
+Starting from Pandas version 1.1.0, the sorting methods accept a ``key``
+argument, so you can simply pass :func:`natsort_keygen` to the sorting
+methods and sort:
+
+.. code-block:: pycon
+
+    >>> df.sort_index(key=natsort.natsort_keygen())
```

Comment:
The `natsort` unit tests run tests on the documentation, so the unit tests need to be self-contained. This will need the import statement and a definition of `df`.

Also, is `sort_index` the best example, or `sort_values`?

Reply:
@SethMMorton , done. But it wants Pandas to be installed.